test/mocks/events/total-scheme-values.js

```diff
@@ -0,0 +1,7 @@
+const { SCHEME_ID } = require('../values/scheme-id')
+const { VALUE } = require('../values/value')
+module.exports = {
```

Comment:
Space needed here

Reply:
also moved this out of the `events` folder as its not an event, doesnt really fit the other folders so put it in the root `mocks` folder